WCCILdata    (0), 2022.02.13 00:42:05.940, SYS,  INFO,        1, Manager Start, PROJ, TEST3.16, V 3.16 - 3.16   platform Windows AMD64 linked at Feb 24 2020 13:28:09 (41a21fb4906)
WCCILdata    (0), 2022.02.13 00:42:05.946, SYS,  INFO,        0, , (Re)Opening databases in single user mode without transaction logging.
WCCILdata    (0), 2022.02.13 00:42:07.030, SYS,  INFO,        0, , Initialize archive smoothing
WCCILdata    (0), 2022.02.13 00:42:07.032, SYS,  INFO,        0, , Initialize archive smoothing
WCCILdata    (0), 2022.02.13 00:42:07.032, SYS,  INFO,        0, , Loading DpIdentification
WCCILdata    (0), 2022.02.13 00:42:07.034, SYS,  INFO,        0, , Initializing last value cache
WCCILdata    (0), 2022.02.13 00:42:07.035, SYS,  INFO,        0, , Loading internal datapoints
WCCILdata    (0), 2022.02.13 00:42:07.035, SYS,  WARNING,    33, General error in Data Manager, DataManDpIntern, loadInternalDp, Internal DPE not found: _DataManager.DataBase.Check:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.035, SYS,  INFO,        0, , Initializing ID history
WCCILdata    (0), 2022.02.13 00:42:07.035, SYS,  INFO,        0, , Initializing Alert history
WCCILdata    (0), 2022.02.13 00:42:07.036, SYS,  INFO,        0, , Initializing Alert Class cache
WCCILdata    (0), 2022.02.13 00:42:07.037, SYS,  INFO,        6, Initialization by Data Manager finished
WCCILdata    (0), 2022.02.13 00:42:07.037, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.arNr:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.037, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.state:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.037, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.general.arType:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.037, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.general.arName:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.037, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.general.hostNameGet:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.037, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.general.hostNameSet:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.general.filePathGet:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.general.filePathSet:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.general.fileNameGet:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.general.fileNameSet:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.general.createArchive:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.general.forwardToRDB:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.error.errNum:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.error.errDp:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.error.errBit:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.error.errRead:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.size.maxDpElGet:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.size.maxDpElSet:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.size.maxValuesGet:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.size.maxValuesSet:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.size.maxFillPctGet:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.size.maxFillPctSet:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.size.maxHeapSizeGet:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.size.maxHeapSizeSet:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.size.maxCacheGet:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.size.maxCacheSet:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.038, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.size.maxCorrCacheGet:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.size.maxCorrCacheSet:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.intervals.aliveInterval:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.intervals.saveCycle:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileSwitch.switchTimeGet.validFrom:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileSwitch.switchTimeGet.validUntil:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileSwitch.switchTimeGet.time:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileSwitch.switchTimeGet.weekDay:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileSwitch.switchTimeGet.monthDay:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileSwitch.switchTimeGet.month:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileSwitch.switchTimeGet.syncTime:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileSwitch.switchTimeGet.syncWeekDay:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileSwitch.switchTimeGet.syncDay:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileSwitch.switchTimeGet.syncMonth:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileSwitch.switchTimeGet.interval:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileSwitch.switchTimeGet.delay:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileSwitch.switchTimeSet.validFrom:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileSwitch.switchTimeSet.validUntil:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileSwitch.switchTimeSet.time:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileSwitch.switchTimeSet.weekDay:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileSwitch.switchTimeSet.monthDay:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.039, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileSwitch.switchTimeSet.month:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.040, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileSwitch.switchTimeSet.syncTime:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.040, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileSwitch.switchTimeSet.syncWeekDay:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.040, SYS,  SEVERE,    118, Last error already repeated 50 times. Until another output occurs, this one will be blocked.
WCCILdata    (0), 2022.02.13 00:42:07.042, PARAM,SEVERE,      8, Element does not exist, problem during archive initialization, datapoint does not exist:, _ValueArchive_0.setMgmt.fileCompression.autoStartAfterSwitchSet:_original.._value
WCCILdata    (0), 2022.02.13 00:42:07.043, SYS,  INFO,      119, Last error repeated 213 times.
WCCILdata    (0), 2022.02.13 00:42:07.043, SYS,  INFO,        0, , Initializing Value Archives
WCCILdata    (0), 2022.02.13 00:42:07.043, SYS,  INFO,        0, , ValueArchive will be used...
WCCILdata    (0), 2022.02.13 00:42:07.466, SYS,  INFO,        4, Connected to (SYS: 1 Event -num 0 CONN: 1) @ DESKTOP-D5C9R5L (::1)
WCCILdata    (0), 2022.02.13 00:42:07.475, SYS,  INFO,        0, , Manager (SYS: 1 Event -num 0 CONN: 1) initialised
WCCILdata    (0), 2022.02.13 00:42:07.714, SYS,  INFO,        4, Connected to (SYS: 1 Driver -num 1 CONN: 1) @ DESKTOP-D5C9R5L (::1)
WCCILdata    (0), 2022.02.13 00:42:07.737, SYS,  INFO,        0, , Manager (SYS: 1 Driver -num 1 CONN: 1) initialised
WCCILdata    (0), 2022.02.13 00:42:08.030, SYS,  INFO,        4, Connected to (SYS: 1 Ascii -num 1 CONN: 1) @ DESKTOP-D5C9R5L (::1)
WCCILdata    (0), 2022.02.13 00:42:08.031, SYS,  INFO,        0, , Manager (SYS: 1 Ascii -num 1 CONN: 1) initialised
WCCILdata    (0), 2022.02.13 00:42:08.351, SYS,  WARNING,    39, Connection lost, MAN: (SYS: 1 Ascii -num 1 CONN: 1), Connection reset by peer (10054)
WCCILdata    (0), 2022.02.13 00:42:08.496, SYS,  INFO,        4, Connected to (SYS: 1 Ascii -num 1 CONN: 1) @ DESKTOP-D5C9R5L (::1)
WCCILdata    (0), 2022.02.13 00:42:08.498, SYS,  INFO,        0, , Manager (SYS: 1 Ascii -num 1 CONN: 1) initialised
WCCILdata    (0), 2022.02.13 00:42:08.830, SYS,  WARNING,    39, Connection lost, MAN: (SYS: 1 Ascii -num 1 CONN: 1), Connection reset by peer (10054)
WCCILdata    (0), 2022.02.13 00:42:09.001, SYS,  INFO,        4, Connected to (SYS: 1 Ascii -num 1 CONN: 1) @ DESKTOP-D5C9R5L (::1)
WCCILdata    (0), 2022.02.13 00:42:09.008, SYS,  INFO,        0, , Manager (SYS: 1 Ascii -num 1 CONN: 1) initialised
WCCILdata    (0), 2022.02.13 00:42:35.468, SYS,  WARNING,    39, Connection lost, MAN: (SYS: 1 Ascii -num 1 CONN: 1), Connection reset by peer (10054)
WCCILdata    (0), 2022.02.13 00:42:37.521, SYS,  INFO,       39, Connection lost, MAN: (SYS: 1 Driver -num 1 CONN: 1), Connection closed
WCCILdata    (0), 2022.02.13 00:42:37.630, SYS,  INFO,       39, Connection lost, MAN: (SYS: 1 Event -num 0 CONN: 1), Connection closed
WCCILdata    (0), 2022.02.13 00:42:39.658, SYS,  INFO,        0, , Loading DpIdentification
WCCILdata    (0), 2022.02.13 00:42:39.718, SYS,  INFO,        2, Manager Stop
